config/clusters/2i2c/mtu.values.yaml

```diff
@@ -23,8 +23,8 @@ jupyterhub:
           name: 2i2c
           url: https://2i2c.org
         funded_by:
-          name: 2i2c
-          url: https://2i2c.org
+          name: ""
+          url: ""
```

Comment:
@lebrown do you have funding info for the hub please?

Reply:
Michigan Technological University - College of Computing
https://www.mtu.edu/computing/

Reply:
Thanks!